plugin/fixmyjs.vim

```diff
@@ -39,6 +54,20 @@ if !exists('g:fixmyjs_rc_filename')
       let g:fixmyjs_rc_filename = 'tslint.json'
   endif
 endif
+if type(g:fixmyjs_rc_filename) == type([])
+  for s:rc_filename in g:fixmyjs_rc_filename
+    let s:paths = s:get_rc_paths(s:rc_filename)
```

Comment:
@fvictorio 
Just got time to test this today, it actually has a logic problem here.

For example, you have `g:fixmyjs_rc_filename = [ '.eslintrc', '.eslintrc.json' ]`, then it will first check `.eslintrc` in path `[s:project_root_path . "/" . a:filename, '$HOME/'.a:filename, '$HOME/.vim/'.a:filename]`. If any of the three path has `.eslintrc` it will just use `.eslintrc`.

In my case, I have `.eslintrc` in my `$HOME` folder, so it always use `$HOME/.eslintrc` no matter if there is `.eslintrc.json` under my project root path.